shared/actions/chat.js

```diff
@@ -745,10 +766,10 @@ function * _loadInbox (action: ?LoadInbox): SagaGenerator<any, any> {
   const conversations: List<InboxState> = _inboxToConversations(inbox, author, following || {}, metaData)
   yield put({type: 'chat:loadedInbox', payload: {inbox: conversations}, logTransformer: loadedInboxActionTransformer})
   chatInboxUnverified.response.result()
+  const selectedConversationIDKey = yield select(_selectedSelector)
 
-  // +1 for the finish call
-  const total = inbox.conversationsUnverified && inbox.conversationsUnverified.length + 1 || 0
-  for (let i = 0; i < total; ++i) {
```

Comment:
This was never reaching the finish call for me. I had to do + 2 to get it to work.